File: APIQuanLyCuaHang/APIQuanLyCuaHang/ViewModels/DataChart/DetailInvoiceDC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace APIQuanLyCuaHang.ViewModels.DataChart
{
    public class DetailInvoiceDC
    {
        public int MaHd { get; set; }

        public int MaCtsp { get; set; }

        public int SoLuong { get; set; }

        public int MaSp { get; set; }
        public int MaDanhMuc { get; set; }

        public string TenSanPham { get; set; } = null!;

        public string? MoTa { get; set; }

        public string? KichThuoc { get; set; }

        public string? HuongVi { get; set; }

        public int? SoLuongTon { get; set; }

        public decimal? DonGia { get; set; }
        public decimal? TongTien { get; set; }
        public string? LinkAnhDau { get; set; }
    }
}

File: QuanLyCuaHangUXUI/QuanLyCuaHangProject/src/models/dto/dataCharts/DetailInvoiceDC.js
class DetailInvoiceDC {
  constructor(
    maHd,
    maCtsp,
    soLuong,
    maSp,
    maDanhMuc,
    tenSanPham,
    moTa = null,
    kichThuoc = null,
    huongVi = null,
    soLuongTon = null,
    donGia = null,
    tongTien = null,
    linkAnhDau = null,
  ) {
    this.maHd = maHd
    this.maCtsp = maCtsp
    this.soLuong = soLuong
    this.maSp = maSp
    this.maDanhMuc = maDanhMuc
    this.tenSanPham = tenSanPham
    this.moTa = moTa
    this.kichThuoc = kichThuoc
    this.huongVi = huongVi
    this.soLuongTon = soLuongTon
    this.donGia = donGia
    this.tongTien = tongTien
    this.linkAnhDau = linkAnhDau
  }
}

export default DetailInvoiceDC
